### ISSUE TITLE
Entities: Incorrect role is defined for ‘Choose column’ control as link, but its behaves like button.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1064]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1261512

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 
Pre-Requisite:
Select "Azure Table" in API combo box while creating the Cosmos DB.
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under  "Azure Table API" Section and select "Entities" which appears after expending the items.
Navigate to 'choose column' control under Advanced options.
Verify the issue.
Actual Result:
Incorrect role is defined for ‘choose column’ control as link, but its behaves like button as dialog box opens after clicking on choose column button.

Expected Result:
Role should be defined as ‘button’ for ‘choose column’ control and appearance of control also should be defined as button, so that user can get correct information about role.

![image](https://user-images.githubusercontent.com/8519407/132710752-c1dce368-e94f-49e7-b5c6-6d214629eac2.png)

**Comment by ato9000** [comment]
AB#1261512
